workers.tf

```diff
@@ -121,3 +122,56 @@ resource "null_resource" "tags_as_list_of_maps" {
     "propagate_at_launch", "true"
   )}"
 }
+
+resource "aws_iam_role_policy_attachment" "workers_autoscaling" {
+  policy_arn = "${aws_iam_policy.worker_autoscaling.arn}"
```

Comment:
How can I have a `count` argument here that is `1` if any group in `worker_groups` map has `autoscaling_enabled` set to true? But 0 otherwise?

Reply:
It's not strictly required as the policy will not allow anything extra due to the conditions anyway.

Reply:
Got it. Sounds like the IAM condition resolves it then.

Re: potential solutions - I too am short on (practical) ideas... if we were able to use `*` against `var.worker_groups` (I don't think this is allowed) and pull the values of `autoscaling_enabled` into a dedicated list, that could be evaluated with `contains()`. A dedicated local or (cringe) output of a submodule might open up the possibility of using `*` against the input in the way we'd need but that's entering **really** hairy territory.